conda_smithy/configure_feedstock.py

```diff
@@ -2456,20 +2456,14 @@ def _load_forge_config(forge_dir, exclusive_config_file, forge_yml=None):
     logger.debug(log)
     logger.debug("## END CONFIGURATION\n")
 
-    if config["provider"]["linux_aarch64"] == "default":
-        config["provider"]["linux_aarch64"] = ["travis"]
+    if config["provider"]["linux_aarch64"] in {"default", "native"}:
```

Comment:
Travis is not native. Use `azure` only for default.

Reply:
Maybe "native" should emit an error for now?